### DISCUSSION TITLE
Photoprism unable to cannot to asustor NAS

### DISCUSSION THREAD

**Discussion by xseph12** [Radiokot/photoprism-android-client#119] [Q&A] [answered]
Hi
I have a problem using photoprism android app.

Photoprism is currently install in my Asustor Nas and i can access the site using my computer browser. But when i try to setup the photoprism in my Android device i encountered this error:
"Library is not accessible:ssl peer unverified exception :..."

I used the address (https://nasip:32770/) which i can open using the computer browser.

Anyone encountered same problem and solve it?

**Comment by Radiokot** [comment, marked as answer]
Hi. Install the root certificate from the NAS on your Android device, issue a valid SSL certificate, or do not use SSL at all. 
While on a computer in a browser it is possible to ignore SSL warnings and proceed with an unverified certificate, it is not possible in the gallery app.
